frontend/src/components/USWDSIcon.tsx

```diff
@@ -12,7 +14,7 @@ const sprite_uri = SpriteSVG.src as string;
 export function USWDSIcon(props: IconProps) {
   return (
     <svg
-      className={props.className}
+      className={clsx("usa-icon", props.className)}
```

Comment:
since all usage of this component had this class added manually, figured we should add it to the shared implementation and make this component a bit more useful